app/src/main/java/com/fstyle/androidtrainning/MainActivity.java

```diff
@@ -1,13 +1,44 @@
 package com.fstyle.androidtrainning;
 
+import android.annotation.SuppressLint;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import org.w3c.dom.Text;
```

Comment:
ERROR: Unused import: org.w3c.dom.Text. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports